inbc-program/README.md

```diff
@@ -114,9 +119,19 @@ inbc sota
      --releasedate 2022-02-22 
      --username <username>
 ```
-#### Edge Device on Ubuntu
+#### Edge Device on Ubuntu in update mode
 ```
-inbc sota
+inbc sota --command update
```

Comment:
Need to put the --command in the correct brackets for whether it is required or optional.  { } is required and [ ] is optional.